db/database.sql

```diff
@@ -1,4 +1,4 @@
-SET NAMES utf8mb4;
+SET NAMES utf8;
```

Comment:
The `utf8mb4` encoding here is required to correctly support many Unicode characters including Emoji, was there a reason you changed this?